src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -163,7 +164,7 @@ void AppCommandlineArgs::_buildParser()
 // - <none>
 void AppCommandlineArgs::_buildNewTabParser()
 {
-    _newTabCommand.subcommand = _app.add_subcommand("new-tab", NEEDS_LOC("Create a new tab"));
+    _newTabCommand.subcommand = _app.add_subcommand("new-tab", winrt::to_string(RS_(L"CmdNewTabDesc")));

```

Comment:
does this seem like an ask for a new macro, `RS_A`?

Reply:
_it kinda does_

Reply:
Works for me.

Reply:
nm had an even crazier and better idea

Reply:
nm the idea was great but the center could not hold